cmd/waku/node.go

```diff
@@ -80,21 +74,37 @@ func scalePerc(value float64) float64 {
 
 const dialTimeout = 7 * time.Second
 
+func nonRecoverErrorMsg(format string, a ...any) error {
+	err := fmt.Errorf(format, a...)
+	return nonRecoverError(err)
+}
+
+func nonRecoverError(err error) error {
+	return cli.Exit(err.Error(), 166)
```

Comment:
So the exit code for this type of failure will be `166`, but I see no change to any documentation or help message to mention this. The `docs/operators/how-to` folder would probably benefit from this information. Maybe a separate file showing an example Systemd service definition.